subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -176,14 +172,14 @@ When configuration cache problems are turned into warnings, Gradle will fail the
 This can be adjusted by specifying an allowed maximum number of problems on the command line:
 
 ----
-❯ gradle -Dorg.gradle.unsafe.configuration-cache.max-problems=5
+❯ gradle -Dorg.gradle.configuration-cache.max-problems=5
 ----
 
 or in a `gradle.properties` file:
 
 [source,properties]
 ----
-org.gradle.unsafe.configuration-cache.max-problems=5
+org.gradle.configuration-cache.max-problems=5
```

Comment:
This gives us the opportunity to make `o.g.configuration-cache.max-problems` and `o.g.configuration-cache-problems` more symmetric by renaming one of them:
- `org.grade.{configuration-cache-problems, configuration-cache-max-problems}`
- `org.gradle.configuration-cache.{problems, max-problems}` 

What do you think?

Reply:
makes sense to me!